setup.py

```diff
@@ -24,7 +24,11 @@
           "Programming Language :: Python",
           "Topic :: Internet :: WWW/HTTP",
           "Topic :: Internet :: WWW/HTTP :: WSGI :: Application",
-          "License :: OSI Approved :: Apache Software License"
+          "License :: OSI Approved :: Apache Software License",
+          'Programming Language :: Python',
```

Comment:
This is duplicated here (thanks for @gabisurita pointing it out in https://github.com/Cornices/cornice.ext.swagger/pull/74#discussion_r149164551)